src/Altinn.Notifications.Core/Services/Interfaces/IOrderRequestService.cs

```diff
@@ -6,12 +6,12 @@ namespace Altinn.Notifications.Core.Services.Interfaces;
 /// <summary>
 /// Interface for the email notification order service
```

Comment:
Comment should be updated to reflect that this does not just support email orders